backend/vouch/views.py

```diff
@@ -25,6 +25,8 @@ class VoucherCreateAPIView(generics.CreateAPIView):
     ),
 )
 class VoucherGivenDestroyAPIView(generics.DestroyAPIView):
+    serializer_class = GivenVoucherSerializer
+
```

Comment:
I find that a bit confusing to set a `serializer_class` on a `DELETE` view that returns an empty 204 response.
Alternatively it seems to be possible to add `responses=None` in  the `extend_schema()` definition:

```suggestion
        responses=None,
    ),
)
class VoucherGivenDestroyAPIView(generics.DestroyAPIView):
```

Reply:
Good, I'll fix this.